index.js

```diff
@@ -0,0 +1,197 @@
+/** @format */
+
+function addItem() {
+  const userInputValue = document.getElementById('input').value;
+
+  const item = {
+    description: userInputValue,
+    completed: false,
+    index: 0,
+  };
+
+  const itemArray = [];
+
+  if (localStorage.getItem('items').length) {
+    const currentItems = JSON.parse(localStorage.getItem('items'));
```

Comment:
- [ ] I suggest you create a function for reading and setting data to local storage to follow the DRY principle and make your code not repeated.

Reply:
This part is a bit confusing as I am setting multiple instances of the items array. Please guide me on this.

Reply:
for example you can use a function for setting items to local storage like this `const setLocalStorage = (tasks) => {
  localStorage.setItem('tasks', JSON.stringify(tasks));
};`

Reply:
- [ ] I suggest you create a function for reading and setting data to local storage to follow the DRY principle and make your code not repeated.